### PR TITLE
test(e2e): extend e2e tests by examples coverage

### PR DESCRIPTION
**Pull request by stupchiy**

### CODE REVIEW

File: test/end-to-end/client.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const Bluebird = require('bluebird');
+
+const path = require('path');
+const { createManagementClient } = require('../../bin/lib/contentful-client');
+
+const config = {
+  accessToken: process.env.CONTENTFUL_INTEGRATION_MANAGEMENT_TOKEN,
+  application: 'contentful.migration-cli.e2e-test/0.0.0'
```
Comment (TimBeyer) [inline test/end-to-end/client.js]: Nice to have a separate useragent thingy here. Maybe we could even `require` the `package.json` and use the version number from there.

File: test/end-to-end/client.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+};
+
+const client = createManagementClient(config);
+
+const makeRequest = function (spaceId, requestConfig) {
+  requestConfig.url = path.join(spaceId, requestConfig.url);
+  return client.rawRequest(requestConfig);
+};
+
+const waitForJobCompletion = Bluebird.coroutine(function * (makeRequest, spaceId, jobId) {
```
Comment (TimBeyer) [inline test/end-to-end/client.js]: Would be cool to also share this code with the integration tests since it's somewhat duplicated

Comment (stupchiy) [reply test/end-to-end/client.js]: good idea, will update

File: test/end-to-end/client.js (hunk 1 above)
Comment (stupchiy) [reply test/end-to-end/client.js]: good idea, will update

**Pull request merged**